locales/id/shipping.ts

```diff
@@ -10,6 +10,7 @@ const locale = {
     "deliveryCourier": "Kurir Pengiriman",
     "inDelivery": "Kurir dalam perjalanan menuju pembeli",
     "inTransit": "Pesanan tiba di",
+    "moreShipping": "Pilih Methode Pengiriman Lainnya",

```

Comment:
```suggestion
    "moreShipping": "Pilih Metode Pengiriman Lainnya",

```

Metode kan ya mestinya?